src/backends/interflop-bitmask/interflop_bitmask.c

```diff
@@ -514,6 +514,20 @@ static void print_information_header(void *context) {
       ctx->daz ? "true" : "false", key_ftz_str, ctx->ftz ? "true" : "false");
 }
 
+void _interflop_enter_function( interflop_function_stack_t *stack,
```

Comment:
Backends that do not use _enter and _exit. Do not need to implement these functions.
Instead they just need to have a NULL entry in the function pointer struct.

This helps keeping backends tidy avoiding empty stubs.

Reply:
They have been replaced by NULL as advised